.github/workflows/staging-to-main.yml

```diff
@@ -0,0 +1,53 @@
+name: Copy staging commits to main
+
+on:
+  push:
+    branches:
+      - 'staging/**'
```

Comment:
https://github.com/OpenSlides/OpenSlides/blob/main/.github/workflows/build-images.yml#L7
runs `on: push: branches: - 'stable/4*'`
And will soon match the `staging` branches as well.
We should keep these matchings uniform. If you're fine with changing this to `'staging/4*'` please do so. Otherwise I will change the matching in `build-images.yml` to `'stable/**'` (and also `'staging/**'` later)

Reply:
Sure, will change